jiant/utils/utils.py

```diff
@@ -355,9 +369,7 @@ def get_elmo_mixing_weights(text_field_embedder, task=None):
 
 def get_batch_size(batch):
     """ Given a batch with unknown text_fields, get an estimate of batch size """
-    batch_field = batch["inputs"] if "inputs" in batch else batch["input1"]
-    keys = [k for k in batch_field.keys()]
-    batch_size = batch_field[keys[0]].size()[0]
+    batch_size = batch["idx"].size()[0]
```

Comment:
This previously supported batches without "idx". I don't think that's a case that we'll actually encounter, but have you double checked that this is safe?

Reply:
Yes, I checked that the current models being supported have "idx" but I'll double check that and also make sure it's in the tutorial.

Reply:
I changed it because the previous way was actually not safe because not all tasks have "inputs" (such as some of the QA models).